README.md

```diff
@@ -42,46 +42,176 @@ These elements are described in different folders. All the modelling happens wit
+```sh
+Installing setuptools, pip, wheel...done.
+Launching subshell in virtual environment. Type 'exit' or 'Ctrl+D' to return.
+openfiscabash-3.2$ 
+```
+Additional information :
+- Exit the virtualenv with `exit` (or Ctrl-D).
+- Re-enter with `pew workon openfisca`.
+
+:tada: You are now ready to install this OpenFisca Country Package!
+
+We offer 2 install procedures. Pick procedure A or B below depending on how you plan to use this Country Package. 
+
+### A. Minimal Installation (Pip Install)
+
+Follow this installation if you wish to:
+- run calculations on a large population;
+- create tax & benefits simulations;
+- write an extension to this legislation (e.g. city specific tax & benefits);
+- serve your Country Package with the OpenFisca Web API.
+
+For more advanced uses, head to the [Advanced Installation](#advanced-installation-git-clone).
+
+#### Install this Country Package with Pip Install
+
+Inside your virtualenv, check the prerequisites:
+
+```sh
+python --version  # should print "Python 2.7.xx".
+#if not, make sure you pass the python version as an argument when creating your virtualenv
+```
+
+```sh
+pip --version  # should print at least 9.0.
+#if not, run "pip install --upgrade pip"
+```
+Install the Country Package:
+
+```sh
+#Example: Openfisca-France
+pip install openfisca-france
+```
+
+:tada: The OpenFisca Country Package is now installed and ready!
+
+#### Next Steps
+
+- To learn how to use OpenFisca, follow our [tutorials](https://doc.openfisca.fr/getting-started.html).
+- To serve this Country Package, serve the [OpenFisca web API](#serve-your-country-package-with-the-openFisca-web-api).
+
+Depending on what you want to do with OpenFisca, you may want to install yet other packages in your virtualenv:
+- To install extensions or write on top of this Country Package, head to the [Extensions documentation](https://doc.openfisca.fr/contribute/extensions.html).
+- To plot simulation results, try [matplotlib](http://matplotlib.org/).
+- To manage data, check out [pandas](http://pandas.pydata.org/).
+
+### B. Advanced Installation (Git Clone)
+
+Follow this tutorial if you wish to:
+- create or change this Country Package's legislation;
+- contribute to the source code.
+
+#### Clone this Country Package with Git
+
+First of all, make sure [Git](https://www.git-scm.com/) is installed on your machine.
+
+Set your working directory to the location where you want this OpenFisca Country Package cloned.
+
+Inside your virtualenv, check the prerequisites:
+
+```sh
+python --version  # should print "Python 2.7.xx".
+#if not, make sure you pass the python version as an argument when creating your virtualenv
+```
+
+```sh
+pip --version  # should print at least 9.0.
+#if not, run "pip install --upgrade pip"
+```
+Clone this Country Package on your machine:
+
+```sh
+#Example: Openfisca-France
+git clone https://github.com/openfisca/openfisca-france.git
```

Comment:
Same: prefer referencing `openfisca-country-template` to benefit from the automating renaming in the bootstraping script.